src/cli/commands/install.js

```diff
@@ -744,7 +744,7 @@ export class Install {
 
   async checkCompatibility(): Promise<void> {
     const {manifest} = await this.fetchRequestFromCwd();
-    await compatibility.checkOne({_reference: {}, ...manifest}, this.config, this.flags.ignoreEngines);
+    await compatibility.checkOne(manifest, this.config, this.flags.ignoreEngines);
```

Comment:
what does this change do? 🤔

Reply:
This was needed to avoid hitting that same invariant that I did.